classes/lmp-machine-custom.bbclass

```diff
@@ -102,6 +102,17 @@ UBOOT_DTB_LOADADDRESS_cl-som-imx7 = "0x82000000"
 UBOOT_RD_LOADADDRESS_cl-som-imx7 = "0x82100000"
 WKS_FILE_sota_cl-som-imx7 = "sdimage-imx7-spl-sota.wks"
 
+# Toradex Apalis iMX6 (support both NAND and eMMC targets with one single image)
+OSTREE_KERNEL_ARGS_append_apalis-imx6 = " console=tty1 console=ttymxc0,115200"
+EXTRA_IMAGEDEPENDS_append_apalis-imx6 = " u-boot-script-toradex"
+KERNEL_DEVICETREE_append_apalis_imx6 = "imx6q-apalis-ixora-v1.1.dtb"
```

Comment:
This shouldn't be needed as the machine configuration file already has such dtb in KERNEL_DEVICETREE.

Reply:
Ok I've dropped this line and the machine conf in favor of the upstream one.